src/components/header/Header.jsx

```diff
@@ -18,7 +18,11 @@ const Header = () => {
           <ButtonPill text="Log in" onClick={logInOnClick} />
         </div>
       ) : (
```

Comment:
Ignore this, it was done in [the previous lesson](https://github.com/kenttk/marks-side-project/pull/26)